spec/spec_helper.cr

```diff
@@ -24,6 +24,14 @@ struct ErrorResponse
   property errors : Array(ErrorData)
 end
 
+def assert_unauthorized!(response : HTTP::Client::Response)
```

Comment:
I don't get why there is an exclamation point. There are usually used to mark a method as "dangerous", like `not_nil!`.

Reply:
This is dangerous method, it raises an exception to fail a test. Regardless, changed.